### ISSUE TITLE
Add more fonts

### ISSUE THREAD

**Issue by typiconman** [typiconman/fonts-cu#47]
The issue with porting other fonts to Unicode is that they were copyrighted by Vlad Dorosh and others, so could not be freely modified.

The fonts in the defunct [CSLTeX project](https://sites.google.com/site/csltex/) are licensed under the LaTeX project license, so they can be freely modified.

Thus, at least the following fonts should be ported and modified to Unicode:
* Triodion UCS
* Starouspenskaya UCS
* Akathistos UCS

Fonts for titling:
* Oglavie
* Kathisma

Fonts for drop caps:
* Vertograd

Anything else?

Pull requests welcome.

**Comment by while0pass** [comment]
Я бы добавил Почаевск -- шрифт с наименьшим размером выносных элементов.

**Comment by typiconman** [comment]
I ported Vlad Dorosh's Oglavie font to Unicode. @starover77 please review.
![oglavie](https://user-images.githubusercontent.com/5428695/56455722-c25bf380-636a-11e9-8665-6a09c783c30e.png)

**Comment by typiconman** [comment]
I ported Vlad Dorosh's Vertograd font to Unicode. @starover77 and others, please review.
![vertograd](https://user-images.githubusercontent.com/5428695/56909656-4e7bc280-6ab1-11e9-89eb-6776891f421a.png)

**Comment by starover77** [comment]
XB!

Oglavie looks great. I think it is ready for public release.

Vertograd.... for what it is, it looks decent, although I personally dislike the design (it doesn't look remotely close to anything "traditionally Slavic" in its artistic style - perhaps pseudo-Gothic?). Setting aside my prejudices... two remarks:

1) When I downloaded the .sfd file and opened it in FontForge, its encoding was set to "custom". It needs to be switched so that it firmly set to Unicode encoding. No big deal there.

2) Can we get rid of that horrid .notdef symbol of a GIANT circle with a slash, and replace it with an "industry-standard" .notdef rectangle which does not exceed the character's line leading?

**Comment by typiconman** [comment]
BB!

Well, Vertograd is the typeface used for initial letters and titling in most Synodal editions published circa 1900, as well as in current reprints.
But sure, I can get rid of the circular .notdef. :)

**Comment by typiconman** [comment]
@starover77 I changed the .notdef / .null glyphs.

**Comment by typiconman** [comment]
I added Cathisma Unicode, a reencoding of Vlad Dorosh's Kathisma UCS font. @starover77 please review.
![cathisma](https://user-images.githubusercontent.com/5428695/57072583-e57d9000-6ce6-11e9-87c1-f38d55bae240.png)

**Comment by starover77** [comment]
Cathisma Unicode looks fine to me.

**Comment by typiconman** [comment]
@while0pass I began adding Pochaevsk (still needs anchor points and validation). @starover77 What do you think about the Typicon symbols? Perhaps we can just import them from Ponomar and tweak the dimensions?
![typiconsymbols_pochaevsk](https://user-images.githubusercontent.com/5428695/60382458-effca180-9a6b-11e9-8214-b306dd07ebbd.png)

**Comment by starover77** [comment]
I believe that the Typicon symbols in Pochaevsk were added from other (dubious) sources. Unless you hear back from me later on the matter, let's just import them from Ponomar.

**Comment by typiconman** [comment]
There are no Typicon symbols in Pochaevsk right now. I'd like to add them in from Ponomar (that's being demonstrated on the image above; sorry that I didn't make it clear). Would you be willing to fine tune the glyphs?

**Comment by starover77** [comment]
A comment on the Slovo Titlo: compared to other diacritical marks, it seems that it is floating too high above the word. I think we should try to make all titla approximately the same height above the characters.

**Comment by typiconman** [comment]
> A comment on the Slovo Titlo: compared to other diacritical marks, it seems that it is floating too high above the word. I think we should try to make all titla approximately the same height above the characters.

Yes, as I wrote above -- there are no anchor points yet. So the diacritical marks are all positioned wrong.

**Comment by starover77** [comment]
I actually have three or four other versions of Pochaevsk which do have some Typicon symbols included. I first received a copy of the font back in 1991 (called ChurchSlavonic, developed by Linotype), and it was I who originally reworked it and called it Pochaevsk in 2000. (I chose that name because Linotype designed the lead type for Jordanville, and later they designed a film and then a digital matrix for Jordanville based on the same design. I was given a copy of that digital matrix. Jordanville's print shop has St. Job of Pochaev as their patron, and is considered to be a legitimate continuation of the Pochaev Monastery printing press, although I understand that the history is a bit complex.)

Anyway, I will look through the other versions of the font and see if I can find the Typicon symbols. I'm not really expecting to find anything that matches the original design of Jordanville's editions, but I can let you know later. I have to leave for work shortly, so I will work on this tonight. If I have to, I will make a fresh design of these characters, as I prefer that we try to match the look of Jordanville's symbols.

**Comment by typiconman** [comment]
Thanks. Didn't realize this font reproduced Jordanville's typeface. Pretty cool!

**Comment by typiconman** [comment]
@starover77 If you have some time, please review. I've added GSUB data and ran the Pochaevsk font through validation algorithms. It may need some fine tuning of anchor points.

**Comment by starover77** [comment]
I've finally had a chance to look at the font in detail, and I see that the outlines are going to need a good deal of cleaning up (it's all minor work, but extensive throughout the font), which I think I can manage to get done on Monday. While I am working on that, I will add the Typicon symbols which I had initially designed to go with this font, based on Jordanville publications.

**Comment by typiconman** [comment]
Triodion Unicode is ready for testing.
The kerning will need some tweaking, and a few glyphs (such as Typicon symbols) need to be added. @starover77 your thoughts?

**Comment by starover77** [comment]
I have a bit of time today to look over the Triodion font, and I will add Typicon symbols, etc. as needed. (I will try to find some time to look over the Smolensky and Shaidur fonts as well, but I may not be able to do much with them today.)

**Comment by while0pass** [comment]
Спасибо, что включили в пакет Почаевск. Было бы замечательно добавить ещё его узкую версию. Гарнитура Почаевск, на мой взгляд, имеет особою типографскую ценность благодаря коротким выносным элементам. Это делает её единственной в своем роде. Она очень практичнапри необходимости вкраплять в текст с гражданским шрифтом. Однако ее классическая версия сохраняет характерную для большинства церковнославянских шрифтов сильную насыщенность, так что на фоне гражданицы церковнославянский текст выглядит полужирным. При черно-белом смешанном наборе (как, например, в церковнославянском словаре ИРЯ РАН, где кириллический и гражданский шрифты соседствуют) это неудобно тем, что резко сокращатеся возможность расставлять визуальные акценты -- полужирный церковнославянский всё перекрывает. Поэтому было бы неплохо иметь версию Почаевска, которая бы несильно по насыщенности отличалась от гражданского текста. В этом случае обычный Почаевск можно использовать в качестве полужирного.

![image](https://user-images.githubusercontent.com/140562/78938256-945f4a00-7aba-11ea-8804-9aba40705c22.png)

К сожалению, в природе нормальной версии такого шрифта не существует. Есть [сплющенная](https://xn--nxa.xn--80adbjsgcdgtdgcxue6e3i.xn--80asehdb/fonts/pochucscond.woff) (событие недопустимое с точки зрения шрифтовиков) версия стандартного Почаевска, которая собственно и используется в церковнославянском словаре. Она визуально адаптирована под насыщенность и убористость гарнитуры Октава. И хотя шрифт, кажется, не потерял лица после механической утрамбовки, как минимум для некоторых надстрочных диакритических букв в нем следовало сохранить прежнюю ширину.

![image](https://user-images.githubusercontent.com/140562/78938303-a7721a00-7aba-11ea-9065-8450faf1446a.png)

Понимаю, что узкий Почаевск выглядит необычно, тем не менее практическая потребность в шрифте с подобными свойствами сохраняется.

**Comment by typiconman** [comment]
@tiroj Perhaps this is an issue you/d be interested in taking up: a condensed version of the Pochaevsk font.

**Comment by tiroj** [comment]
Possibly. I'll take a look when I have some time. Things just got very busy.

**Comment by typiconman** [comment]
I ported Akafistos to Unicode (I called it Acathist). @starover77 I think it is more or less ready for testing.
![Screenshot from 2020-04-13 08-13-04](https://user-images.githubusercontent.com/5428695/79094063-ee803980-7d5e-11ea-8a9b-be81e5ef9020.png)

**Comment by KrasnayaPloshchad** [comment]
It would be nice if you get permissions to convert more fonts from irmologion.ru

**Comment by typiconman** [comment]
> It would be nice if you get permissions to convert more fonts from irmologion.ru

All fonts that are part of the [CSLTeX](https://sites.google.com/site/csltex/) package can be converted since they are licensed under the LaTeX license.
Pull requests are welcome.

**Comment by KrasnayaPloshchad** [comment]
They should also have instructions on documentation as well.

**Comment by typiconman** [comment]
For the documentation, pull requests are welcome.

**Comment by EliseyP** [comment]
Простите, это к вопросу выше - можно ли в будущем перевести в Unicode декоративный шрифт Bukvica UCS? Вот например файл шрифта:
[https://github.com/EliseyP/csl_odt2tex/blob/main/Bukvica.ttf](https://github.com/EliseyP/csl_odt2tex/blob/main/Bukvica.ttf).

**Comment by typiconman** [comment]
@EliseyP можно, если это позволяет его лицензия. Кто владеет авторскими правами?

**Comment by EliseyP** [comment]
Постараюсь узнать в ближайшее время. Очень уж хорошие буквицы.

**Comment by EliseyP** [comment]
Похоже, что шрифт свободен.
https://cocpucm.livejournal.com/64662.html?ysclid=ll8lbaf2ms772146844
